ticketutil/servicenow.py

```diff
@@ -365,6 +364,14 @@ def _prepare_ticket_fields(self, fields):
         return fields
 
 
+def devops_one_url(server, table, sys_id):
```

Comment:
One more quick question on this - Should this helper function really be in ticketutil? Or should it be in your external code that calls ticketutil? Would anyone else find this function useful, or is it really just for your specific use case of ServiceNowTicket? 

Just trying to think about this from a general perspective - I don't want any Red Hat Internal functions in ticketutil if they wouldn't benefit anyone outside of Red Hat.